apis/nucleus/src/viz.js

```diff
@@ -237,6 +283,34 @@ export default function viz({
       await rendered;
       return cellRef.current.takeSnapshot();
     },
+    // ===== undocumented experimental API - use at own risk ======
+    /**
+     *  valid types: viewData, cssScaling, snapshot, exportData, exploration
+     *  questionable types: supportRefresh, quickMobile, fullscreen
+     *  deprecated?: sharing
+     *
+     */
+    support(type) {
+      if (type === 'viewData' && viewDataObjectId !== undefined) {
+        return true;
+      }
+      const renderState = cellRef.current.getRenderState();
+      // Could be cleaner, but need to check that its rendered properly
+      if (mountedReference && renderState.loaded && !renderState.error && successfulRender) {
+        const supportObject = cellRef.current.getExtDefinition()?.support;
+        if (supportObject) {
+          // Passing in layout here should only be relevant if the chart has rendered
+          return support(type, supportObject, model?.layout);
```

Comment:
Info:
for model.layout to work it needs enigma.js mixins (added by qlik-embed / sense-client)
but if used in context without the mixins model.layout will always be undefined.

Reply:
I can move it into Cell, that makes it simpler